docs/API/fliplet-ui-toast.md

```diff
@@ -105,6 +105,32 @@ If a progress bar is present, sets the progress bar to the given progress. The p
 Fliplet.UI.Toast('App updated');
 ```
 
+### Use a Toast notification to show error messages
+
+```js
+// Data Source 0 is not found, and will trigger an error
+Fliplet.DataSources.connect(0)
+  .then(funciton (connection) {
+    return connection.find();
+  })
+  .catch(function (error) {
+    console.error(error);
```

Comment:
I'd remove the console.error since people might think it's related or required to using the toast